test/kuttl/tests/glance_dist_image_import/01-import-image.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - script: |
+        oc -n $NAMESPACE exec -it openstackclient --stdin=false -- /bin/sh -c "DOMAIN=glance-default-single.$NAMESPACE.svc ./scripts/dist-image-import.sh"
```

Comment:
Should we copy the script before to pod?

Reply:
It's already there because the `configMapGenerator` builds the configMap that is already mounted in the `openstackclient` `Pod`. 
This way you only need to call the script.